data/helpers.d/mysql

```diff
@@ -90,3 +90,47 @@ ynh_mysql_create_user() {
 ynh_mysql_drop_user() {
     ynh_mysql_execute_as_root "DROP USER '${1}'@'localhost';"
 }
+
+# Create a database, an user and its password. Then store the password in the app's config
+#
+# User of database will be stored in db_user's variable.
+# Name of database will be stored in db_name's variable.
+# And password in db_pwd's variable.
```

Comment:
These lines are not clear to me : there's no `db_user` or `db_name` variable ? Were these supposed to be returned value somehow ?